test/tutorial/ssft_simple_2.c

```diff
@@ -1,5 +1,10 @@
-// RUN: %shorntest %t-harness.ll %t-debug %s | OutputCheck %s
+// RUN: %shorntest %t-harness.ll %t-debug %s --max-depth=9 | OutputCheck %s --check-prefix=XFAIL_UNKNOWN
+// RUN: %shorntest %t-harness0.ll %t-debug0 %s -O0 --max-depth=9 | OutputCheck %s
+// RUN: %shorntest %t-harness1.ll %t-debug1 %s -O1 --max-depth=9 | OutputCheck %s --check-prefix=XFAIL_UNKNOWN
+// RUN: %shorntest %t-harness2.ll %t-debug2 %s -O2 --max-depth=9 | OutputCheck %s --check-prefix=XFAIL_UNKNOWN
+// RUN: %shorntest %t-harness3.ll %t-debug3 %s -O3 --max-depth=9 | OutputCheck %s --check-prefix=XFAIL_UNKNOWN
```

Comment:
need to add `--enable-indvar` for which the program produces `unknown`.

Actually, need to check  with `--enable-indvar` on all other examples. It is ok if it returns `unknown` but it is not ok when the option switches `sat` to `unsat` and vice versa